<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodificadorDlg</class>
 <widget class="QDialog" name="CodificadorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="placeholderText">
    <string>CODIFICADOR DE RESISTENCIAS</string>
   </property>
  </widget>
  <widget class="KColorButton" name="kcolorbutton5">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>140</y>
     <width>33</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="KColorButton" name="kcolorbutton1">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>140</y>
     <width>33</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="KColorButton" name="kcolorbutton3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>140</y>
     <width>33</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="color">
    <color>
     <red>0</red>
     <green>0</green>
     <blue>0</blue>
    </color>
   </property>
  </widget>
  <widget class="KColorButton" name="kcolorbutton2">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>140</y>
     <width>33</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="KColorButton" name="kcolorbutton4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>33</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio4B">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>105</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>4 Bandas</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio5B">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>105</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>5 Bandas</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="botonCodificar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="valorResistencia">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
